.github/CODEOWNERS

```diff
@@ -0,0 +1,26 @@
+# https://help.github.com/articles/about-codeowners/
+
+# These owners are the default owners for all Unikraft core files.
+*                     @unikraft/maintainers-core
+
+# Along with all default owners, these owners oversee hardware architectures.
+/arch/                @unikraft/maintainers-arch
```

Comment:
There aren't teams named `maintainers-arch`, `maintainers-plat`, `maintainers-libs` yet.

Reply:
Hi @razvand,

These lists will be populated automatically by a "governance" repository defining who is in these lists via a YAML file.